PopcornEditor.js

```diff
@@ -10,9 +10,30 @@ function PopcornEditor() {
 
   this.init = function (el, url) {
     var editor = (typeof el === 'string') ? document.getElementById(el) : el,
-        url = url || 'PopcornEditor/editor.html';
+        url = url || 'PopcornEditor/editor.html',
+        loadingDiv = null,
+        loadingSpinner = null;
 
     window.addEventListener('message', onmessage);
+    this.listen('loaded', function () {
+      document.querySelector('#loading').remove();
+    });
+
+    loadingDiv = document.createElement('div');
+    loadingDiv.setAttribute('id', 'loading');
+    loadingDiv.style.width = '100%';
+    loadingDiv.style.height = '100%';
+    loadingDiv.style.backgroundColor = 'white';
+    loadingDiv.style.display = 'flex';
+    loadingDiv.style.justifyContent = 'center';
+
+    loadingSpinner = document.createElement('img');
+    loadingSpinner.setAttribute('class', 'loading-spinner');
+    loadingSpinner.setAttribute('src', 'PopcornEditor/resources/ring-alt.svg');
```

Comment:
Yeah, I'm a bit confused. This is how I see it atm:

* PopcornEditor loads an `iframe`
* In the `iframe` is the actual editor, *which includes* `PopcornEditor/resources/ring-alt.svg`.
* This code reaches into the src that's meant for the content *inside* the `iframe` to get an image.

If the stuff inside `src` is meant to be its own beast, I'd put this image outside in the context of `PopcornEditor.js`.

Otherwise, ¯\_(ツ)_/¯ sure! This makes sense.